packages/@o3r/core/schematics/sdk/rules/clean-standalone.rule.ts

```diff
@@ -21,3 +21,30 @@ export function cleanStandaloneFiles(targetPath: string): Rule {
     return tree;
   };
 }
+
+function getRelativePath(input: string): string {
+  const depth = input.split('/').filter(segment => segment !== '').length;
+  return new Array(depth).fill('..').join('/');
+}
+
+/**
+ * Replace files generated by the standalone SDK generator
+ * @param targetPath Path where the SDK has been generated
+ */
+export function replaceStandaloneFiles(targetPath: string): Rule {
+  return (tree) => {
+    tree.overwrite(posix.join(targetPath, 'jest.config.js'), `const getJestConfig = require('${getRelativePath(targetPath)}/jest.config.ut').getJestConfig;
```

Comment:
could make more sense to be in a `templates` folder which make it easier to maintain